utils/complianceFlow.ts

```diff
@@ -4,7 +4,7 @@ import { Deployment, Version } from 'types/interfaces'
 
 function approvalColour(state: string) {
   if (state === 'No Response') return yellow['200']
-  if (state === 'Accepted') return green['A200']
+  if (state === 'Accepted') return green.A200
   if (state === 'Declined') return red['200']
```

Comment:
Is there a reason we're now using a different format for green and red values here?